libs/application/templates/driving-assessment-approval/src/fields/StudentLookupField.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FC } from 'react'
+import { useQuery, gql } from '@apollo/client'
+import { useWatch } from 'react-hook-form'
+import { CustomField, FieldBaseProps } from '@island.is/application/core'
+import { Box, Text } from '@island.is/island-ui/core'
+import * as kennitala from 'kennitala'
+
+const QUERY = gql`
+  query studentInfo($nationalId: String!) {
+    drivingLicenseStudentInformation(nationalId: $nationalId) {
+      student {
+        name
+      }
+    }
+  }
+`
+
+interface Props extends FieldBaseProps {
+  field: CustomField
+}
+
+export const StudentLookupField: FC<Props> = ({ error }) => {
+  const studentNationalId = useWatch({
+    name: 'student.nationalId',
+  })
+
+  const { data = {}, error: queryError, loading } = useQuery(QUERY, {
+    skip:
+      !studentNationalId || !kennitala.isPerson(studentNationalId as string),
+    variables: {
+      nationalId: studentNationalId,
+    },
+  })
+
+  if (queryError) {
+    return <Text>Villa kom upp við að sækja upplýsingar um nemanda</Text>
```

Comment:
Consider moving this string to messages to support multiple languages.
This applies to all unlocalized strings